modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/challenge/questions/mgt/ChallengeQuestionManagementAdminServiceTestCase.java

```diff
@@ -53,7 +54,7 @@
     private static final String SUPER_TENANT = "carbon.super";
     private static final String WSO2_TENANT = "wso2.com";
 
-    private static final String LOCALITY_CLAIM_URI = "http://wso2.org/claims/locality";
+    private static final String LOCALITY_CLAIM_URI = IdentityUtil.getClaimUriLocale();
```

Comment:
I don't think from tests we can access IS runtime data. Did you run the test with this change?

Reply:
yes, i ran the tests